Client portion of trip updates feed

unreasonableman opened the pull request:
Tested against trip inference training data set.

unreasonableman committed:
- don't skip last entry in training data set when computing results
- add start and end date for a specific error message

unreasonableman committed:
total wip update, incomplete and untested

unreasonableman committed:
first bit of very superficial testing, and resulting bug fixes..

unreasonableman committed:
fix some bugs around stop-time-entities generation

unreasonableman committed:
- fix bug that retrieved only the first character of assigned trip ID from metadata.txt
- fix multiple bugs in util.getproperty()
- more in-depth testing

unreasonableman committed:
fix time offset bug, rename some offset variables for clarity

unreasonableman committed:
send stop time entities from baby shark if available

unreasonableman committed:
at least be consistent with '-' vs '_' within the same message

unreasonableman committed:
fix property name

owades commented on graas-bt.py:
```diff
@@ -246,138 +220,6 @@ def read_gps_data(ser):
 
     return gps
 
-def handleButton(text):
```
I'm seeing a lot of code removed here without being replaced elsewhere - just double checking that that's ok

unreasonableman committed:
- consistent return value types for graas-bt.send_at()
- consistent return values for inference.get_trip_id()
- add -u option to graas-bt.py to set server url

unreasonableman commented:
Yup, it's the bluetooth UI which we're not really using anymore.

On Mon, May 16, 2022 at 4:24 PM Scott Owades ***@***.***>
wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In graas-bt.py
> <https://github.com/cal-itp/graas-hardware/pull/7#discussion_r874233262>:
>
> > @@ -246,138 +220,6 @@ def read_gps_data(ser):
>
>      return gps
>
> -def handleButton(text):
>
> I'm seeing a lot of code removed here without being replaced elsewhere -
> just double checking that that's ok
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/cal-itp/graas-hardware/pull/7#pullrequestreview-974666956>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAGLQDOSBFMUFBMGBFXC2ETVKLKKBANCNFSM5WCIOKBA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
